life/modernized/05-chapter-4.adoc

```diff
@@ -39,6 +39,8 @@ to reap and to gather in.`"
 After describing the field of labour, into which several of these brethren had entered,
 he continues,
 
+[quote]
+____
```

Comment:
This also appears to be from a letter to Edward to Margaret Fell, so [.embedded-content-document.letter]
The same changes will need to be done in the original too. 👍🏻